galaxy_ng/app/migrations/0018_set_rate_limit_default.py

```diff
@@ -0,0 +1,24 @@
+from django.db import migrations
+
+def set_rate_limit_default(apps, schema_editor):
+    CollectionRemote = apps.get_model('ansible', 'CollectionRemote')
+
+    # By default pulp core sets the new rate_limit field to null, which remotes the
```

Comment:
s/remotes/removes

Curious, if the user sets the rate limit to zero, does that also effectively remove it?

Reply:
Setting it to 0 does remove the rate limit